tasks/sysctl.yml

```diff
@@ -5,7 +5,7 @@
     owner: 'root'
     group: 'root'
     mode: '0440'
-    state: touch
+    state: file
```

Comment:
You will run into errors with this if the file does not exist (ansible file module errors out when`state: file` is used on a non-existent path). The proper way to fix idempotence is

```yaml
state: touch
modification_time: preserve
access_time: preserve
```

https://docs.ansible.com/ansible/latest/modules/file_module.html

Reply:
That is why it was touch, not file! Thanks! That's what happens if you don't document your code.

I'll fix thix.